config/routes.rb

```diff
@@ -1,8 +1,19 @@
 Rails.application.routes.draw do
-  get 'users/index'
-  get"top/index" => "top#index"
-  get 'home/index'
+  root "top#index"
+  get "courses" => "courses#index"
+  post "courses" => "courses#create"
+  get "new_course" => "courses#new"
```

Comment:
下記のほうが他と統一感があるので良さそうです
```suggestion
  get "courses/new" => "courses#new"
```